oat-sa/extension-tao-outcomeui
# Feature/nsa 113/new column is correct

https://oat-sa.atlassian.net/browse/NSA-113

Show correct responses / Show if given responses are correct

How to test:
1. Pass a test
2. Got to Export table page
3. Generate a preview or generate a csv with result
4. Check a new column `Item #-RESPONSE_is_correct`
